src/account/management/commands/createpublicanonuser.py

```diff
@@ -0,0 +1,104 @@
+import factory
+from django.db.models import signals
+
+from django.core.management.base import BaseCommand
+from django.utils.crypto import get_random_string
+from django.contrib.auth.models import User
+
+from config import settings
+from core.db.manager import DataHubManager
+
+
+class Command(BaseCommand):
+    help = ("Creates public an anonymous users that are "
+            "necessary for publishing data")
+
+    def handle(self, *args, **options):
+        create_public_user(None, None)
+        create_anonymous_user(None, None)
+
+
+@factory.django.mute_signals(signals.pre_save)
+def create_public_user(apps, schema_editor):
+    # Create public user
+    username = settings.PUBLIC_ROLE
+    email = settings.PUBLIC_ROLE_EMAIL
+    # generate a password for the public user
```

Comment:
If you create the Django user first and pass no password, has_usable_password() will be False so no one will ever be able to log in as that user. Also, its password attribute will be set to something random which you can use later when creating a db role.